check/tests/pass.rs

```diff
@@ -346,10 +348,10 @@ test
     let result = support::typecheck(text);
     assert!(result.is_ok(), "{}", result.unwrap_err());
 
-    let variants = Type::variants(vec![(support::intern_unscoped("T"),
+    let variants = Type::variants(collect![(support::intern_unscoped("T"),
                                         Type::function(vec![typ("a")],
                                                        support::typ_a("Test", vec![typ("a")])))]);
```

Comment:
Ihave got rustfmt set to format automatically but I think `vec!` is special cased, other macros do not get formatted...